lib/utilities/detect-deprecated-config.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var glob = require('glob');
+
+module.exports = function() {
+  var deprecatedConfigPath = path.join(__dirname, '../../../..', 'config', 'deploy', '*.js');
```

Comment:
Not sure if there is a better way to get to the user's home directory or not but also, shouldn't this be in the following format to handle different platform path spacing (not sure if that matters in our current scenario or not)

```
path.join(__dirname, '..', '..', '..', '..', 'config', 'deploy', '*.js');
```